tests/mbed/host/host.c

```diff
@@ -91,7 +91,7 @@ void Test(oe_enclave_t* enclave, int selftest, char* data_file_name)
     }
 }
 
-OE_OCALL void ocall_exit(uint64_t arg)
+void ocall_exit(int arg)
```

Comment:
I am surprised that the call to exit(uint64_t) did not generate a warning (as error).
Maybe we are relaxing the warning level for this test since it is 3rd party?